README.md

```diff
@@ -205,6 +207,31 @@ describe( "duck reducer", function( ) {
 } );
 ```
 
+### Typescript
+Your interface and type definitions must also be a part of your duck interface. I consider keeping them with the source generally a good practice. In order to allow import destructuring your duck must export them as named exports from index.ts. I usually find myself having interface definitions in three places: actions.ts (action interfaces), reducers.ts (state interfaces) and selectors.ts (selector interfaces). 
+Your duck might end up looking like this:
+```javascript
+import reducer from "./reducers";
+
+import * as duckSelectors from "./selectors";
+import * as duckOperations from "./operations";
+import * as duckActionTypes from "./action-types";
+import * as duckActions from "./actions";
+
+export {
+    duckActions,
+    duckSelectors,
+    duckOperations,
+    duckActionTypes
+};
+
+export { IActionInterface1, IActionInterface2 } from "./actions";
```

Comment:
I don't think this syntax works, or at least it's not part of the standard ESModules spec

Reply:
Beg to differ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
haven't played with this in a while, will try it out